components/motor/ulnstepper/28byj-48.go

```diff
@@ -243,6 +256,30 @@ func (m *uln28byj) setPins(ctx context.Context, pins [4]bool) error {
 	return err
 }
 
+func (m *uln28byj) getTargetStepPosition() int64 {
+	m.lock.Lock()
+	defer m.lock.Unlock()
+	return m.targetStepPosition
+}
+
+func (m *uln28byj) setTargetStepPosition(targetPos int64) {
+	m.lock.Lock()
+	defer m.lock.Unlock()
+	m.targetStepPosition = targetPos
+}
+
+func (m *uln28byj) getStepPosition() int64 {
+	m.lock.Lock()
+	defer m.lock.Unlock()
+	return m.stepPosition
+}
```

Comment:
`Position` should probably be called this and divided by the step position.

Also - I don't think the step position should be configurable for this motor now that I think about it, I think this driver can only move this particular motor, lol. That's fr another day/pr.

Reply:
To be clear, not blocking.

Reply:
Merge whenever.